projects/com.oracle.truffle.llvm.nodes.impl/src/com/oracle/truffle/llvm/nodes/impl/func/LLVMGlobalRootNode.java

```diff
@@ -104,13 +104,16 @@ protected Object executeIteration(VirtualFrame frame, int iteration, Object[] ar
         if (printExecutionTime) {
             startExecutionTime = System.currentTimeMillis();
         }
+
+        executeConstructorFunctions();
```

Comment:
We execute the constructor functions the first time in the [LLVM class](https://github.com/graalvm/sulong/blob/6c78c9b8afad7a5fb1440959c43812af4ff40ab3/projects/com.oracle.truffle.llvm/src/com/oracle/truffle/llvm/LLVM.java#L189) since they also need to be executed when a function is called over interop where this node is not constructed. I think we should not change this logic in this node.